zookeeper-server/pom.xml

```diff
@@ -77,6 +77,7 @@
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-log4j12</artifactId>
+      <scope>test</scope>
```

Comment:
slf4j-log4j12-1.7.35 maven coordinates automatically redirect to slf4j-reload4j-1.7.35

So, you could just use the slf4j-reload4j binding directly, if you update the slf4j version to the latest, 1.7.35 (which I would recommend). If you do this, I see this would be done in the root POM, plus the server, plus 3 of the contrib POMs.

See https://search.maven.org/artifact/org.slf4j/slf4j-log4j12/1.7.35/pom